src/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -50,15 +50,56 @@ public enum GCNotificationStatus
         NotApplicable = 4
     }
 
+    [StructLayout(LayoutKind.Sequential)]
+    public readonly struct GCMemoryInfo
+    {
+        private readonly int m_highMemoryLoadThreshold;
```

Comment:
Nit: We prefer a simple `_` prefix in C#. It is fine to use `_` on the unmanaged side as well. The C/C++ coding style in VM is much less uniform.